src/runtime/analytics-service.js

```diff
@@ -267,11 +267,16 @@ export class AnalyticsService {
   }
 
   /**
-   * Returns true if any logs have already be sent to the analytics server.
+   * Returns true if the user was sent to gPay (and not redirected).
    * @return {boolean}
    */
-  getHasLogged() {
-    return this.everStartedLog_;
+  getWaitingOnGpay() {
+    return this.waitingOnGpay_;
+  }
+
+  /** */
```

Comment:
Do you need this line? If you do It is nicer if you add some description even if it is mostly restating what can be deduced from the name of the function